app/presenters/exception_hunter/error_group_presenter.rb

```diff
@@ -11,7 +11,8 @@ def self.wrap_collection(collection)
     end
 
     def self.format_occurrence_day(day)
-      day.to_date.strftime('%A, %B %d')
+      date = day.to_date
+      date == Date.yesterday ? 'Yesterday' : date.strftime('%A, %B %d')
```

Comment:
😄 Could you write a test for this change?

Reply:
Done!